.claude/skills/splitting-cards/references/nid-inheritance.md

```
@@ -27,34 +23,73 @@ await ankiConnect('updateNoteFields', {
+  fields: { Text: card.content, 'Back Extra': '' },
+  tags: [...originalTags, ...card.inheritTags],
+}));
+const results = await addNotes(notes);
 ```
 
 ## 역링크 형식
 
-새 카드에 원본 카드로의 nid 링크 삽입:
+새 카드에 원본 카드로의 nid 링크 삽입. `createBackLink()` (`nid-parser.ts`):
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)